src/components/Tutorial.jsx

```diff
@@ -0,0 +1,175 @@
+			],
+		},
+		{
+			step: 3,
+			imgLink: '/assets/img/tutorial/tut-2.png',
+			text: [
+				`Dit zijn de instellingen van de Brainweb-hulpfuncties. `,
+				`Hier kan je de functies gebruiken die wij speciaal voor jou hebben gemaakt. `,
+				`Om bepaalde functies te gebruiken, moet je eerst inloggen met je Brainweb-account. `,
+			],
+		},
+		{
+			step: 4,
+			imgLink: '/assets/img/tutorial/scrollhulp.png',
+			text: [
+				`De scrollhulp biedt hulp op de pagina's die je bezoekt.`,
+				`Als je deze functie aanzet en je opent een webpagina, zal er rechts onderaan het scherm een vierkantje verschijnen 
+        met 2 pijlen en een sleep-symbool in het midden.`,
+        `Je kan de scrollsnelheid ook aanpassen door op de plus -en minknop te klikken.`,
+				`Als je over 1 van deze pijlen zweeft met je muis, zal de pagina omhoog of omlaag worden gescrolld. Als je klikt, zal 
+        de pagina wat sneller scrollen. `,
+			],
+		},
+		{
+			step: 5,
+			imgLink: '/assets/img/tutorial/radial-ui.png',
+			text: [
+				`De radial-ui zal ervoor zorgen dat als je rechts klikt op je muis, een cirkelvormig menu verschijnt met een aantal 
+        opties.`,
+			],
+		},
+		{
+			step: 6,
+			imgLink: '/assets/img/tutorial/tracking.png',
+			text: [
+				`De muis-tracking zal muisgegevens verzamelen op websites die je bezoekt.`,
+				`Deze gegevens worden gebruikt om voorspellingen te maken over waar je naartoe zou willen met je muis. In de volgende stap 
+        lees je meer over de voorspellingen.`,
+        `Zoals er naast de knop staat, moet je wel inloggen om dit te kunnen gebruiken. Dit omdat we de gegevens moeten opslaan 
+        in onze database en je hier een account voor nodig hebt.`,
+			],
+		},
+		{
+			step: 7,
+			imgLink: '/assets/img/tutorial/predictions.png',
+			text: [
+				`Zoals in vorige stap werd gezegd, kan je hier de voorspellingen aan of uit zetten. Er wordt voorspeld waar je naartoe
+        zou willen gaan met je muis en dan worden bepaalde elementen dichter bij de muis getrokken of vergroot.`,
+			],
+		},
+    {
+      step: 8,
+      imgLink: '/assets/img/logo-brainweb.png',
+      text: [
+        `Tot zo ver de rondleiding. Aarzel niet om ons feedback te geven in de instellingen van de browser!`,
+        `Wij hopen alvast dat je de browser nuttig vindt en dat het je kan helpen bij het surfen op het web.`,
+        `Tot slot, bedankt voor je interesse in onze browser!`,
+      ],
+    },
+	];
+
+	return (
+		<div
+			id="tutor"
+			className="absolute left-0 right-0 top-0 bottom-0 z-[999] m-auto flex h-[48rem] w-[35rem] flex-col rounded-xl bg-skin-dark p-5 pb-7"
+		>
+			<div className="flex flex-grow flex-col">
+				<div className="w-full">
+					<XIcon
+						className="float-right w-8 cursor-pointer text-[#cbd5e1]"
+						onClick={() => {
+							setTutorial(false);
+						}}
+					/>
+				</div>
+				<div className="h-[17rem] mb-2">
+					<img src={`${content[step - 1].imgLink}`} className="mx-auto object-cover" />
```

Comment:
Ik zou currentContent in een variable bijhouden. Dan moet je niet altijd `content[step-1]` doen